slackevents/inner_events.go

```diff
@@ -649,6 +649,51 @@ type SharedInvite struct {
 	IsExternalLimited bool            `json:"is_external_limited,omitempty"`
 }
 
+// FunctionExecutedEvent is sent if a function was executed - https://api.slack.com/events/function_executed
+type FunctionExecutedEvent struct {
+	Type                string            `json:"type"`
+	Function            Function          `json:"function"`
+	Inputs              map[string]string `json:"inputs"`
+	FunctionExecutionID string            `json:"function_execution_id"`
+	WorkflowExecutionID string            `json:"workflow_execution_id"`
+	EventTs             string            `json:"event_ts"`
+	BotAccessToken      string            `json:"bot_access_token"`
+}
+
+// Function is a struct for functions in FunctionExecuted events
+type Function struct {
```

Comment:
Where did you find the documentation for `FormEnabled` and `DateReleased`? I don't see it on the [event documentation page](https://api.slack.com/events/function_executed).

Reply:
I got it from the debug mode of the slack-go socket mode, because before that I had not found the official documentation
<img width="1281" alt="image" src="https://github.com/user-attachments/assets/c06d64ea-019f-4273-90bb-0ec65c939f6f">
https://gist.github.com/yogasw/65d68c081bfd5bea14520cedc907a979